modules/src/script/setupModules.groovy

```diff
@@ -24,6 +28,7 @@ println "[INFO] Using HV version " + hvArtifactName;
 processFileInplace( hvModuleXml ) { text ->
     text.replaceAll( /hibernate-validator.*jar/, hvArtifactName )
 }
+appendDependency( hvModuleXml, "javax.money.api", true )
```

Comment:
I may misremember, but didn't we want to add this optional dependency to the descriptor to begin with? Then a user would only have to drop in the modules (and add the API module in jboss-deployment-structure.xml), but not fiddle with any existing module.xml.